Dockerfile

```diff
@@ -14,11 +14,11 @@ RUN wget https://dl.dropboxusercontent.com/u/4550074/mesos/mesos-dns+50fc45a9 -O
 ADD ./isolator /isolator/isolator/
 ADD ./m4 /isolator/m4/
 ADD ./bootstrap /isolator/
-ADD ./calico_isolator.py /isolator/
+ADD ./calico_isolator /isolator/
 ADD ./configure.ac /isolator/
 ADD ./Makefile.am /isolator/
 ADD ./requirements.txt /isolator/
-ADD http://downloads.mesosphere.io/demo/calico-node/v0.5.1/calico-node-v0.5.1.tar /isolator/
+COPY ./calico-node-v0.5.4.tar /isolator/
```

Comment:
Makefile works. We would eventually need to put all calico-specific bits into a calico directory, so that the rest of the repo is pretty much independent of it.